packages/babel-core/src/transformation/internal-plugins/block-hoist.js

```diff
@@ -1,7 +1,6 @@
-import Plugin from "../plugin";
 import sortBy from "lodash/sortBy";
 
-export default new Plugin({
```

Comment:
They still need to be `Plugin` instances eventually, but now I'm just relying on the standard plugin initialization interface via https://github.com/babel/babel/pull/5489/files#diff-d6903991e1c21aaacc4d615157f587e5R30 instead of manually calling `new Plugin`.